test/sim/abilities/sturdy.js

```diff
@@ -56,4 +56,39 @@ describe('Sturdy', function () {
 		battle.makeChoices('move sleeptalk', 'move fusionflare');
 		assert.fainted(battle.p1.active[0]);
 	});
+
+	it.skip(`should trigger before Focus Sash`, function () {
```

Comment:
The order on cart for that event:
1. Endure
2. False Swipe / Hold Back
3. Sturdy
4. Focus Sash / Focus Band

False Swipe should be in here; I have no idea what Bide is doing here, it doesn't belong. Disguise also doesn't belong (should be earlier), but it does at least avoid the problem of Focus Sash triggering before it or something.

Reply:
Wait. so if you use False Swipe against a foe that's using Endure then the cart announces that the foe endured the hit? That's weird...

Disguise is in my list just because breaking Disguise successfully causes zero damage before Focus Sash can intervene, and Bide is in my list just because the amount of damage recorded by Bide is after the reduction from the above effects.

Reply:
> Wait. so if you use False Swipe against a foe that's using Endure then the cart announces that the foe endured the hit? That's weird...

Yep, that's right.